java-symbol-solver-core/src/test/resources/javaparser_expected_output/com_github_javaparser_ast_comments_CommentsParser.txt

```diff
@@ -12,7 +12,7 @@
   Line 51) CommentsCollection comments = new CommentsCollection() ==> com.github.javaparser.ast.comments.CommentsCollection
   Line 51) new CommentsCollection() ==> com.github.javaparser.ast.comments.CommentsCollection
   Line 52) int r ==> int
-  Line 54) Deque prevTwoChars = new LinkedList<Character>(Arrays.asList('z', 'z')) ==> java.util.Deque
+  Line 54) Deque prevTwoChars = new LinkedList<Character>(Arrays.asList('z', 'z')) ==> java.util.Deque<java.lang.Object>
```

Comment:
Probably not, we should add it. Unless we can consider that when there are no parameters is implictly a raw reference